phpstan.neon.dist

```diff
@@ -0,0 +1,16 @@
+includes:
+    - phpstan-baseline.neon
+parameters:
+    level: 0
+
+    paths:
+        - src
+    excludes_analyse:
+        # Class Symfony\Component\Form\Extension\Core\ChoiceList\SimpleChoiceList not found. I can't even find this class too
```

Comment:
https://github.com/sonata-project/SonataAdminBundle/issues/4484

Reply:
thanks for the reference. What do you want me to do with this?

Reply:
IMHO we could remove the class
- ModelChoiceList
- LegacyModelsToArrayTransformer
Since we don't support symfony 2.7

I would consider this as BC. What do you think ?